### PR TITLE
Add me to team.json

### PR DESCRIPTION
**Pull request by johnflux**

### CODE REVIEW

**Review by 0x22almostEvil — CHANGES_REQUESTED** [review]
You also need to add yourself here:
```json
    {
      "name": "ML engineers",
      "members": ["theblackcat102", "sanagno", "johnflux"]
    },
```

**Comment by 0x22almostEvil** [comment]
> You also need to add yourself here:
> 
> ```json
>     {
>       "name": "ML engineers",
>       "members": ["theblackcat102", "sanagno", "johnflux"]
>     },
> ```

Or to full stack, not sure

**Review by olliestanley — APPROVED** [review]

File: website/src/data/team.json
Comment (0x22almostEvil) [inline website/src/data/team.json:1]: nice

**Review by 0x22almostEvil — APPROVED** [review]

**Review by yk — APPROVED** [review]
thank you

**Review by notmd — APPROVED** [review]
Thanks

**Pull request merged**